contributors/design-proposals/architecture/namespaces.md

```diff
@@ -83,7 +83,7 @@ and reference particular entities across operations.
 A *Namespace* provides an authorization scope for accessing content associated
 with the *Namespace*.
 
-See [Authorization plugins](../admin/authorization.md)
+See [Authorization plugins](../auth/access.md)
```

Comment:
This should be linking to https://kubernetes.io/docs/admin/authorization/

Reply:
fixed

Reply:
fixed